pkg/migrations/sql/versions/dev/0.3.1-dev/3-metric_metadata_objects.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS SCHEMA_CATALOG.metadata
+(
+    id SERIAL NOT NULL PRIMARY KEY,
```

Comment:
do we need an id here?

Reply:
I kept it for easier reference but you are right, we do not need this.